test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -9,6 +9,37 @@
 import org.junit.Test;
 
 public class UtilsTest {
+    
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
There are other methods apart from `Utils.isAnyNull` that you could use and I would encourage you to explore them and use them